src/v1/atm/atm-dao.js

```diff
@@ -0,0 +1,15 @@
+import BaseDao from '../../lib/base-dao-mongoose';
+import Atm from './atm-model';
+
+class AtmDao extends BaseDao {
+  async findInArea(latitude, longitude, radius) {
+    return Promise.resolve(
+      this.Model.find({
```

Comment:
This is what I'm talking about -> this.model.find() looks much better and that's the typical use case, so do make Model lowercase